generators/client-2/templates/src/main/webapp/app/account/register/_register.html

```diff
@@ -117,8 +117,9 @@ <h1 translate="register.title">Registration</h1>
                 <button type="submit" [disabled]="registerForm.form.invalid" class="btn btn-primary" translate="register.form.button">Register</button>
             </form>
             <p></p>
-            <div class="alert alert-warning" translate="global.messages.info.authenticated" translate-compile>
-                If you want to <a class="alert-link" (click)="openLogin()">sign in</a>, you can try the default accounts:<br/>- Administrator (login="admin" and password="admin") <br/>- User (login="user" and password="user").
+            <div class="alert alert-warning">
+                <span jhi-translate="global.messages.info.authenticated.prefix">If you want to </span>
+                <a class="alert-link" (click)="openLogin()">sign in</a><span jhi-translate="global.messages.info.authenticated.suffix">, you can try the default accounts:<br/>- Administrator (login="admin" and password="admin") <br/>- User (login="user" and password="user").</span>
```

Comment:
Don't u need to put the sign in text in a span inside the anchor and apply translate on that as well, same in other places